src/opers.h

```diff
@@ -214,7 +215,33 @@ static inline Int ENABLED_ATTR(Obj oper)
 */
 static inline void SET_ENABLED_ATTR(Obj oper, Int x)
 {
-    OPER(oper)->enabled = INTOBJ_INT(x);
+    Obj val = CONST_OPER(oper)->enabled;
+    Int v = val ? INT_INTOBJ(val) : 0;
+    OPER(oper)->enabled = INTOBJ_INT(v | 1);
```

Comment:
So, this abuses `enabled` as a bit array. Since it now stores two bits (not just one), perhaps we should rename it. But I couldn't think of a good name -- my only idea, `flags`, already is used by another field in struct `OperBag`...